### ISSUE TITLE
PHS Data Update Q4

### ISSUE THREAD

**Issue by jesse-WSDOT** [wsdot-eso/Environmental-Information#178]
**Description: PHS Data Update Q4**
**Category: Workbench**
**Workspace: I:\DATA\ExternalData\DFW\PHS\Update20210915**
**% Complete: 25** %

PHS Data received from DFW on 9/15/2021
Ready to load to 17P_EDIT

**Comment by kechinn** [comment]
Please make sure metadata shows updated email and removed phone #

**Comment by gfuellingwa** [comment]
Waiting on DFW for clarification on Marbled murrelet table to join to Occurrence point data.

**Comment by gfuellingwa** [comment]
Ready for ETL. Metadata updated(checked contact info for correct email) and spreadsheet updated.

**Comment by gfuellingwa** [comment]
From Lori:

I’ve attached the ws_mmoccurobs_sv point database instead of the relate table. We figured out that anytime over the last few months that we did a statewide request that our script was not providing the statewide relate table ws_murreletobs_v, not sure why at this point, so far you are the only one who has noticed or at least asked about it. I would ignore the marbled murrelets in the ws_occurpoint_dr or delete them and just use the point layer. Sorry for the inconvenience this may have caused you or the new point layer might cause you. I think once we get some new habitat data imported into our scripts I would like to drop the table, delete the murrelets from ws_occurpoint_dr and just provide the ws_mmoccurobs_sv point layer. It’s so much easier to work with than setting up a relate table. Murrelets are no longer considered sensitive data in our policy – 5210. Thanks for letting us know they were missing from our data we send out.
